web/app/file/relayfile.module.css

```
@@ -409,13 +446,23 @@
 }
 
 @keyframes floatPrimary {
```

Stylelint (17.11.0)

[error] Expected keyframe name "floatPrimary" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)